src/raw/bson.rs

```diff
@@ -610,20 +610,11 @@ impl<'a> TryFrom<RawBson<'a>> for Bson {
 /// A BSON binary value referencing raw bytes stored elsewhere.
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub struct RawBinary<'a> {
```

Comment:
I updated this type to match the regular `Binary`, which has all `pub` fields. Note that this isn't `non_exhaustive` (neither is `Binary`).